docs/components/Select/Web.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Option, Select } from "@jobber/components/Select";
+import { Divider } from "@jobber/components/Divider";
+import { Content } from "@jobber/components/Content";
+
+export default {
+  title: "Components/Selections/Select/Web",
+  component: Select,
+  parameters: {
+    viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@jobber/components/Select": ["Select", "Option"],
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof Select>;
+
+const BasicTemplate: ComponentStory<typeof Select> = args => (
+  <Select {...args}>
+    <Option value="tony">Tony</Option>
+    <Option value="quincy">Quincy</Option>
+    <Option value="peppa">Peppa Pig</Option>
+  </Select>
+);
+
+const SizesTemplate: ComponentStory<typeof Select> = args => {
+  return (
+    <Content>
+      <Select size="small">
```

Comment:
Clever & well done way to handle the args here for this "sticker sheet" type presentation! :)